docs/source/EccFramework.rst

```diff
@@ -0,0 +1,154 @@
+
+  result = qecc.apply_ecc(file, ecc, ecc_frequency)
+
+  circ = QuantumCircuit().from_qasm_str(result["circ"])
+  print(circ)
+
+Currently, the error correction schemes Q3Shor, Q5Laflamme, Q7Steane, Q9Shor, Q9Surface, and Q18Surface are supported.
+
+We provide a wrapper script for applying error correction to quantum circuits (provided as OpenQASM) and followed by a noise-aware quantum circuit simulation (using Qiskit). The script can be used like this:
+
+.. code-block:: file
+
+  $ ecc_qiskit_wrapper -ecc Q7Steane -fq 100 -m D -p 0.0001 -n 2000 -fs aer_simulator_stabilizer -s 0 -f  ent_simple1000_n2.qasm
+  _____Trying to simulate with D(prob=0.0001, shots=2000, n_qubits=17, error correction=Q7Steane) Error______
+  State |00> probability 0.515
+  State |01> probability 0.0055
+  State |10> probability 0.0025
+  State |11> probability 0.477
+
+The script offers a help function, which displays available parameters:
+
+.. code-block:: console
+
+  $ ecc_qiskit_wrapper --help
+  usage: ecc_qiskit_wrapper [-h] [-m M] [-p P] [-n N] [-s S] -f F [-e E] [-fs FS] [-ecc ECC] [-fq FQ] [-mc MC] [-cf CF]
+
+  QiskitWrapper interface with error correction support!
+
+  optional arguments:
+    -h, --help  show this help message and exit
+    -m M        Define the error_channels (e.g., -m APD), available errors channels are amplitude damping (A), phase flip (P), bit flip (B), and depolarization (D) (Default="D")
+    -p P        Set the noise probability (Default=0.001)
+    -n N        Set the number of shots. 0 for deterministic simulation (Default=2000)
+    -s S        Set a seed (Default=0)
+    -f F        Path to a OpenQASM file
+    -e E        Export circuit, with error correcting code applied, as OpenQASM circuit instead of simulation it (e.g., -e "/path/to/new/openqasm_file") (Default=None)
+    -fs FS      Specify a simulator (Default: "statevector_simulator" for simulation without noise, "aer_simulator_density_matrix", for deterministic noise-aware simulation"aer_simulator_statevector", for stochastic noise-
+                aware simulation). Available: [AerSimulator('aer_simulator'), AerSimulator('aer_simulator_statevector'), AerSimulator('aer_simulator_density_matrix'), AerSimulator('aer_simulator_stabilizer'),
+                AerSimulator('aer_simulator_matrix_product_state'), AerSimulator('aer_simulator_extended_stabilizer'), AerSimulator('aer_simulator_unitary'), AerSimulator('aer_simulator_superop'),
+                QasmSimulator('qasm_simulator'), StatevectorSimulator('statevector_simulator'), UnitarySimulator('unitary_simulator'), PulseSimulator('pulse_simulator')]
+    -ecc ECC    Specify a ecc to be applied to the circuit. Currently available are Q3Shor, Q5Laflamme, Q7Steane, Q9Shor, Q9Surface, and Q18Surface (Default=none)
+    -fq FQ      Specify after how many qubit usages error correction is applied to it (Default=100)
+
+Available error-correcting codes
+#####################################################
```

Comment:
I am fairly certain that headline is too long as well. Please look through the RtD build log as suggested above.

Reply:
Changed it to: 
"Supported ECCs"
"========================"